tests/onnx/test_onnx_v2.py

```diff
@@ -337,3 +337,46 @@ def test_tensorflow_export_seq2seq_with_past(
         self, test_name, name, model_name, feature, onnx_config_class_constructor
     ):
         self._onnx_export(test_name, name, model_name, feature, onnx_config_class_constructor)
+
+
+class LayerTestCase(TestCase):
+    """Tests export of specific Layers / Modules."""
+
+    @require_torch
+    @pytest.mark.filterwarnings("ignore:.*Dropout.*:UserWarning:torch.onnx.*")  # torch.onnx is spammy.
+    def test_StableDropout(self):
+        """Tests export of StableDropout in training mode."""
+        import os
+
+        import torch
+
+        from transformers.models.deberta import modeling_deberta
```

Comment:
I would make those imports at the top of the file under the `if torch_available()`.

Reply:
Done.